packages/web/utils/dateHelpers.ts

```diff
@@ -23,7 +23,7 @@ export const getPlayerTimeZoneDisplay = (
 ): TimeZoneDisplay => {
   let tzLabel;
   let offsetLabel;
-  const timeZone = TimeZoneOptions.find((t) => t.name === zone);
+  const timeZone = null; // TimeZoneOptions.find((t) => t.name === zone);
```

Comment:
Do all the timezone selectors work again? I assume we want to reinstate react-timezone since it didn't fix the FOUC?